src/Razor/src/Microsoft.VisualStudio.RazorExtension/VSPackage.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="101" xml:space="preserve">
+    <value>IDS_EDITORNAME</value>
```

Comment:
Should this be something human readable?

Reply:
Eh it probably could be but in the EditFactory docs they use this name pretty religiously so I wanted to stay consistent with what the docs mention.